Doc/conf.py

```diff
@@ -69,10 +70,11 @@
 version, release = importlib.import_module('patchlevel').get_version_info()
 
 rst_epilog = f"""
+.. |archives_version| replace:: {version if 'daily' in tags else release}
 .. |python_version_literal| replace:: ``Python {version}``
 .. |python_x_dot_y_literal| replace:: ``python{version}``
 .. |usr_local_bin_python_x_dot_y_literal| replace:: ``/usr/local/bin/python{version}``
-"""
+"""  # noqa: F821
```

Comment:
`tags` is injected into the `conf.py` namespace: https://www.sphinx-doc.org/en/master/usage/configuration.html#project-tags